content/en/docs/multi-center/modules/hosp/emar.md

```diff
@@ -0,0 +1,72 @@
+---
+title: "emar"
+linktitle: "emar"
+weight: 8
+date: 2023-09-18
+description: >
+  The Electronic Medicine Administration Record (eMAR); barcode scanning of medications at the time of administration.
+---
+
+## *emar*
+
+The *emar* table is used to record administrations of a given medicine to an individual patient.
+Records in this table are populated by bedside nursing staff scanning barcodes associated with the medicine and the patient.
+
+## Links to
+
+* *prescriptions* on `pharmacy_id`
```

Comment:
remind me to double check this, I thought it would be `poe_id` but the docs say `pharmacy_id`